test/test.js

```diff
@@ -89,26 +89,6 @@ test('Auth succesful', t => {
   })
 })
 
-test('Auth not succesful', t => {
```

Comment:
Why this test has been removed?

Reply:
was not meant to, thanks for pointing out